src/lib/updateGrantsRecipients.js

```diff
@@ -432,6 +436,32 @@ export async function processFiles(hashSumHex) {
 
       await Promise.all(grantReplacementPromises);
 
+      // ---
+      // Update GroupGrants
+      const HOUR_AGO = new Date(new Date() - 60 * 60 * 1000);
+
+      const replacements = await GrantReplacements.findAll({
+        where: { updatedAt: { [Op.gte]: HOUR_AGO } },
+        attributes: ['replacedGrantId', 'replacingGrantId'],
+        raw: true,
+      });
+
+      const affectedGroupGrants = await GroupGrant.findAll({
+        where: { grantId: uniq(replacements.map((g) => g.replacedGrantId)) },
+        attributes: ['id', 'grantId', 'groupId'],
+        raw: true,
```

Comment:
it's not a criticism but a question: I'm curious as to why we're transforming these to raw JSON, given it costs some cycles and I can't see why?

Reply:
hm yeah, I don't think I have a good reason for doing this! I probably set it for less verbose output when debugging something and forgot to pull it out :)

removed here: 587675310